resip/dum/ServerAuthManager.cxx

```diff
@@ -148,7 +148,17 @@ ServerAuthManager::handleUserAuthInfo(UserAuthInfo* userAuth)
    {
       InfoLog (<< "Error in auth procedure for " << userAuth->getUser() << " in " << userAuth->getRealm());
       auto response = std::make_shared<SipMessage>();
-      Helper::makeResponse(*response, *requestWithAuth, 503, "Server Error.");
+
```

Comment:
I don't see the value in providing 2 code options here, I would remove the C++17 version here for simplicity.

Reply:
Thanks, I removed the macro RESIP_CPP_STANDARD (I've added it in another PR) and the C++ 17 version methods to simplify things.